appveyor.yml

```diff
@@ -38,8 +38,8 @@ deploy:
       secure: GJVmu2j1exMpBljCZYYxq0OfX2Ug2TK/GM50LxmJg02jGRVmJJ98kGT/aMoNAwxC
     skip_symbols: false
     artifact: /.*(\.|\.s)nupkg/
-    on: 
-      APPVEYOR_REPO_TAG: true        # deploy on tag push only
+#    on: 
```

Comment:
NB for @viceroypenguin: don't forget to uncomment these tags before approving PR